spec/controllers/events_controller_spec.rb

```diff
@@ -217,33 +217,33 @@
     it "should return an pdf with the eating habits of the user" do
       login(:organizer)
       event = Event.create! valid_attributes
-      
+
       user = FactoryGirl.create(:user)
       profile = FactoryGirl.create(:profile, user: user, last_name: "Peter")
-      application_letter = FactoryGirl.create(:application_letter_accepted, 
+      application_letter = FactoryGirl.create(:application_letter_accepted,
         user: user, event: event, vegan: true)
       user = FactoryGirl.create(:user)
       profile = FactoryGirl.create(:profile, user: user, last_name: "Paul")
-      application_letter = FactoryGirl.create(:application_letter_accepted, 
+      application_letter = FactoryGirl.create(:application_letter_accepted,
```

Comment:
My editor (Atom) automatically deletes trailing whitespaces in files I touch. Is that a bad thing?